src/Mono.Android/Android.Runtime/JNIEnv.cs

```diff
@@ -29,7 +29,9 @@ struct JnienvInitializeArgs {
 		public int             localRefsAreIndirect;
 		public int             grefGcThreshold;
 		public IntPtr          grefIGCUserPeer;
-		public int             isRunningOnDesktop;
+		public byte            isRunningOnDesktop;
+		public byte            brokenExceptionTransitions;
+		public PackageNamingPolicy packageNamingPolicy;
```

Comment:
As this struct is shared with C++/`libmonodroid.so`, I think this field should properly be an `int`, and then we cast in `JNIEnv.Initialize()`.  The cast should be optimized away to ~nothing.